Dockerfile.rocm.ubi

```diff
@@ -238,15 +238,14 @@ ENV HF_HUB_OFFLINE=1 \
     # Silences the HF Tokenizers warning
     TOKENIZERS_PARALLELISM=false  \
     RAY_EXPERIMENTAL_NOSET_ROCR_VISIBLE_DEVICES=1 \
-    FLASH_ATTENTION_USE_TRITON_ROCM="TRUE" \
+    HIP_FORCE_DEV_KERNARG=1 \
```

Comment:
Performance flag, see

https://rocm.docs.amd.com/en/docs-6.2.0/how-to/tuning-guides/mi300x/workload.html#vllm-performance-optimization